syntax = "proto3";

message Login{
	string name = 1;
	string password = 2;
}
message Answer{
    bool is_auth = 1;
}

message Authorize{
    string user_id = 1;
    int32 vehicle_id = 2;
}
message Finish{
    string user_id = 1;
    int32 vehicle_id = 2;
    float tempo = 3;
}
service Authentication{
    rpc makelogin(Login) returns (Answer){}
    rpc authorize_vehicle(Authorize) returns (Answer) {}
    rpc user_finish(Finish) returns (Answer) {}
}
